### PR TITLE
added all files

### PR DESCRIPTION
**Pull request by SotoArmando**
New Pull request n.n

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    margin: 0;
+}
+section.hero img {
+    width:100%;
+    position: absolute;
+    z-index: -1;
+    margin-top: -39.4rem;
+}
+section.hero p {
+    text-align: center;
```
Comment (azdrenymeri) [inline css/main.css:25]: `text-align: center;` is also declared on line 30

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    color: white;
+    
+    font-size: 33px;
```
Comment (azdrenymeri) [inline css/main.css:27]: `font-size` property is overridden by line 32

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
# Nice Work
### Design looks good
Here are some fixes you need to make 
- The name of the project is too long and unnecessary so don't include your name and the Microverse on the project name it should be simple.
- I see only one collaborator do you have a coding partner? 
- There is  only one commit make sure on your next project to commit more often(is suggested after every feature)
-  Update the README.md file, include a small description(explain the purpose of the project), the contributor's links and the live version if it is available.
- check https://validator.w3.org/#validate_by_input and fix the markup errors on your index.html file

**Comment by SotoArmando** [comment]
Gotcha! thanks am making the changes n.n.

**Comment by SotoArmando** [comment]
Alright , Changes done!  (I do have a partner but this project is individual)
Indeed i wasn't able to add titles as first child of sections, as it will make the look different from the main site. so just let me know if i need to add them and i will.

**Comment by azdrenymeri** [comment]
# Well Done 🥇 
### No more changes required from my side

**Comment by azdrenymeri** [comment]
> Alright , Changes done! (I do have a partner but this project is individual)
> Indeed i wasn't able to add titles as first child of sections, as it will make the look different from the main site. so just let me know if i need to add them and i will.

No need

**Pull request merged**